### ISSUE TITLE
[Feature Request] Ubisoft - Free Games on Giveaways, etc

### ISSUE THREAD

**Issue by sealldeveloper** [vogler/free-games-claimer#61]
As the title says.

**Comment by vogler** [comment]
Do you have some examples?
https://free.ubisoft.com/ currently only lists game trials, free to play and free weekends - nothing to claim to keep.

**Comment by sealldeveloper** [comment]
In some cases Ubisoft can giveaway games to keep. I don't think an example exists currently but in the past some Assassins Creed and old Tom Clancy games were given away to keep. I only recall finding them in the top-bar of the Ubisoft Connect app, the right-most item.

**Comment by vogler** [comment]
Seems like there are giveaways, but not very often. Can check for the next one if it's also listed on https://free.ubisoft.com/ and under which label.

Also found these lists:
https://gg.deals/news/free-uplay-games/
https://www.gamerpower.com/giveaways/ubisoft/free-games
(https://www.mydealz.de/search?q=ubisoft - search for 'kostenlos')

**Comment by sealldeveloper** [comment]
I'll keep an eye out and if one comes up ill check if its published on that URL and what label it is. 👍 

(The one I was trying to remember was Splinter Cell!)

**Comment by EQUALIT-CG** [comment]
@sealldeveloper 

Yes you're right they give away of "Assassins Creed and old Tom Clancy games" when Tom Clancy rainbow 6 first came out.
Because they needed player and they were also trying to complete with xbox, playstation etc etc

they many do it at xmaxs but as far as i know they don't do it same as EA Origin they also don't really do it anymore they do from time to time but most of them are just demos

**Comment by Primajin** [comment]
What about the free to play ones, would it be worth it?

<img width="250" alt="Screenshot 2023-06-02 at 18 22 38" src="https://github.com/vogler/free-games-claimer/assets/1742115/5e501bf0-b355-4a62-b42d-e283ffb3527e">

**Comment by sealldeveloper** [comment]
> What about the free to play ones, would it be worth it?
> <img alt="Screenshot 2023-06-02 at 18 22 38" width="250" src="https://user-images.githubusercontent.com/1742115/242939204-5e501bf0-b355-4a62-b42d-e283ffb3527e.png">

I don't think so, free isn't changing to paid anytime soon and I think it might just clog libraries.

**Comment by Primajin** [comment]
Alrighty - got it, so then only games that would normally cost money but are free for a promotion.

**Comment by sealldeveloper** [comment]
> Seems like there are giveaways, but not very often. Can check for the next one if it's also listed on https://free.ubisoft.com/ and under which label.
> 
> Also found these lists: https://gg.deals/news/free-uplay-games/ https://www.gamerpower.com/giveaways/ubisoft/free-games (https://www.mydealz.de/search?q=ubisoft - search for 'kostenlos')

Also I took a look at the web archive between dates of previous promotions. https://free.ubisoft.com/ does not seem to showcase Splinter Cell [between the dates is was being promoted](https://web.archive.org/web/20221123125428/https://free.ubisoft.com/).

But looking at [the homepage around that time](https://web.archive.org/web/20221119232245/https://www.ubisoft.com/en-us/), there seems to be a box up the top used for it, but its also utilized for non-claimable games. [Like at the moment](https://web.archive.org/web/20230603210421/https://www.ubisoft.com/en-us/).

Filtering by the `Claim Now` text seems to be the most consistent, I checked [another promo](https://web.archive.org/web/20230603210421/https://www.ubisoft.com/en-us/) and it also had the same text.

**Comment by sealldeveloper** [comment]
An example for the [Rabbid's Promos store page](https://web.archive.org/web/20230119154752/https://store.ubi.com/us/rayman-raving-rabbids-tm-/56c4948888a7e300458b47de.html?lang=en_US) is also archived.

**Comment by vogler** [comment]
@sealldeveloper Thanks!
`a[aria-label="Claim Now"]` would probably be a good selector.

<img width="1134" alt="image" src="https://github.com/vogler/free-games-claimer/assets/493741/012c6d0d-878d-4d70-a54d-16829bb005f3">

**Comment by yoshimo** [comment]
There are also some goodies like Demos  for Monopoly or Prince of Persia that are for free all the time.
